do-catalog/src/components/ModalSubscription.vue

```diff
@@ -1,63 +1,111 @@
+  >
     <div @click="closeModal()" class="close-modal">
       <img src="../assets/modal/close.svg" alt="close" />
     </div>
 
-    <div class="grid u-flex__justify--center u-mt--120">
-      <div class="grid-cell--col6 u-align--center">
-        <img src="../assets/modal/data-request.svg" alt="Request data" />
-        <h2 class="title is-sectiontitle is-txtNavyBlue u-mt--24">
-          Confirm your request
-        </h2>
-        <p class="text is-caption is-txtNavyBlue u-mt--12">
-          You are going to request a subscription to the following dataset and
-          its geography.
-        </p>
-        <p class="title is-small is-txtNavyBlue u-flex u-mt--48">
+    <div class="grid u-flex__justify--center">
+      <div class="grid-cell--col6">
+        <div class="u-align--center">
           <img
-            class="u-mr--8"
-            src="../assets/modal/db-icon.svg"
-            alt="Dataset selected"
+            v-if="getHeaderIcon"
+            :src="require('../assets/modal/' + getHeaderIcon)"
+            alt="Request data"
           />
-          1 Dataset
+          <h2 class="title is-sectiontitle is-txtNavyBlue u-mt--24">
+            {{ getTitle }}
+          </h2>
+          <p class="text is-caption is-txtNavyBlue u-mt--12">
+            {{ getSubTitle }}
+          </p>
+        </div>
+        <ul>
+          <DatasetListItem
+            :key="dataset.slug"
+            :dataset="dataset"
+          ></DatasetListItem>
+        </ul>
+
+        <p
+          v-if="currentMode === 'request'"
+          class="text is-caption is-txtNavyBlue u-mt--16 u-align--center"
+        >
+        Once you confirm your request, a
+          <span class="is-semibold">CARTO team member will get in touch</span>
+          to give you more information and go over any questions you may have.
         </p>
-        <ul class="u-mt--10 data-request-container">
-          <li class="u-flex data-request-card u-mb--24">
-            <div class="card-info u-mr--16">
-              <h4 class="text is-caption is-txtNavyBlue">{{ dataset.name }}</h4>
-              <p
-                class="text is-small is-txtNavyBlue is-semibold u-mt--4 u-flex"
-              >
-                <img
-                  class="u-mr--8"
-                  src="../assets/modal/provider.svg"
-                  alt="Provider"
-                />
-                {{ dataset.provider_name }}
-              </p>
-            </div>
-            <div
-              class="dataset-type-container"
-              :class="{ 'public-data': dataset.is_public_data }"
-            >
-              <span class="dataset-type title">{{ datasetPrivacy }}</span>
-            </div>
+
+        <p
+          v-else-if="currentMode === 'unsubscribe'"
+          class="text is-caption is-txtNavyBlue u-mt--16"
+        >
+        If you unsubcribe pellentesque diam nisi, faucibus varius enim mollis sit amet. Cras nec varius magna, in dignissim diam:
```

Comment:
Wise words :joy:

Reply:
Jejeje I forgot to mention this in the PR comments. We should decide the final text